db/helpers/common.js

```diff
@@ -188,7 +189,18 @@ function publishChange(inst, event, changedKeys, ignoreAttributes) {
   }
 
   if (obj[event]) {
-    pub.publish(channelName, JSON.stringify(obj));
+    const objectAsString = JSON.stringify(obj);
+    zlib.deflate(objectAsString, (err, zippedValue) => {
+      if (err) {
+        console.log('Error deflating!', err);
```

Comment:
if we get an error deflating, shouldn't we try to publish the event normally, rather than just give up and "return"?